schedule/migration/offline_regression_upgrade@pvm.yaml

```diff
@@ -54,9 +54,6 @@ conditional_schedule:
         - console/zypper_lifecycle
         - console/orphaned_packages_check
         - console/consoletest_finish
-        - x11/desktop_runner
```

Comment:
In fact, you don't need remove these modules from this yaml file which used for spvm migration test. For this ticket, it only focus on the removal of x11 modules from ppc64le_regression_test_offline_textmode.yaml

Reply:
Ok, we have the same issue for powerVM, so NP.